pkg/reporting/cookbooks.go

```diff
@@ -113,10 +114,10 @@ func (crs CookbookRecordsByNameVersion) Swap(i, j int) {
 	crs[i], crs[j] = crs[j], crs[i]
 }
 func (crs CookbookRecordsByNameVersion) Less(i, j int) bool {
-	if crs[i].Name != crs[j].Name {
-		return crs[i].Name < crs[j].Name
+	if strings.ToLower(crs[i].Name) != strings.ToLower(crs[j].Name) {
```

Comment:
As a performance improvement, you could create two variables with the lower case to avoid the second time you lower it.